### DISCUSSION TITLE
Mobile: Keyboard input lags

### DISCUSSION THREAD

**Discussion by Minecodes** [ppy/osu#21823] [Q&A]
### Type

Performance

### Bug description

When I press the "x" key on my tablet keyboard (Samsung Galaxy Tab S8+ with the keyboard case directly attached and with a normal USB keyboard), it has an estimated delay of over 250 ms to be intercepted.

### Screenshots or videos

https://user-images.githubusercontent.com/65566012/209705486-19adda02-e094-406c-b9b0-04eff6c94c15.mp4

### Version

2022.1208.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10310040/database.log)
[network.log](https://github.com/ppy/osu/files/10310041/network.log)
[performance.log](https://github.com/ppy/osu/files/10310042/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10310044/runtime.log)

**Comment by Skibidi1927** [comment]
I have this issue too while playing mania with usb keyboard it delay 250ms - 400ms

**Reply by Minecodes** [reply]
I have similar delays myself, but still so severe that I don't hit the notes properly
